dialogue-core/src/main/java/com/palantir/dialogue/core/RetryingChannel.java

```diff
@@ -339,8 +339,7 @@ private void logRetry(long backoffNanoseconds, @Nullable Throwable throwable) {
                         SafeArg.of("backoffMillis", TimeUnit.NANOSECONDS.toMillis(backoffNanoseconds)),
                         SafeArg.of("channelName", channelName),
                         SafeArg.of("serviceName", endpoint.serviceName()),
-                        SafeArg.of("endpoint", endpoint.endpointName()),
-                        throwable);
```

Comment:
what's happening here?

Reply:
So these stacktraces are actually always identical (they don't include the cause), and they made the SimulationTest logs impossible to read. I'll put it back to keep this PR focussed, but I think it's worth considering eliding them.